src/index.ts

```diff
@@ -100,18 +98,9 @@ interface QueryResult {
   rows?: QueryResultRow[]
 }
 
-type ExecuteAs = 'array' | 'object'
-
-type ExecuteOptions = {
```

Comment:
I believe we still want to keep this because we want to limit the number of valid options passed into `Execute`.